### PR TITLE
Example of how the codeblock can be used

### PR DESCRIPTION
**Pull request by kallewesterling**
This is just an example of how the codeblock can be used to highlight features of a certain code, and I think it would be great if we could do something like this throughout. It will ease the transition into color-coded code blocks on the website in a later stage too.

### CODE REVIEW

File: lessons.md [hunk 1]
```diff
@@ -12,11 +12,15 @@
...
-```
 In HTML: 
+
+```html
 <h1> Heading 1 </h1>
+```
 
 In Markdown:
+
+```
```
Comment (kallewesterling) [inline lessons.md:23]: actually, this line should be

` ```md `

**Comment by hackettka** [comment]
i did go back and do this throughout.. are you not seeing it?

File: lessons.md (hunk 1 above)
Comment (hackettka) [reply lessons.md:23]: are you saying i need to ad 'md' after the hashes thorughout? and if so, is it
```md
code block
```md

or 

```md
code block
```

or something else?

**Comment by hackettka** [comment]
When I look at the file here, it looks exactly like i want it to: https://github.com/DHRI-Curriculum/git/blob/v2.0-kristen-edits/lessons.md

But maybe you're telling me to add md after the top ``` for future styling you're planning? 

The example you use changes how I want that part to display so i'm going to leave that alone for now.

**Comment by kallewesterling** [comment]
OK - we can look at it later. The styling is happening already, in the ingestion to the database, and your question makes me think that I need to work a little more on documenting what is happening between what you (as fellows) do in the DHRI GitHub repositories, and what I do on the website.. I'll work on that before this sprint review!

**Pull request closed without merging**